rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/ContextMenu.java

```diff
@@ -131,6 +131,24 @@ public static Menu createContextMenuFromCompactView(RMController controller, Obj
         exportNodeSourceSubItems.setItems(exportNodeSourceToFileItem, exportNodeSourceToCatalogItem);
         exportNodeSourceItem.setSubmenu(exportNodeSourceSubItems);
 
+        MenuItem exportInfrastructureItem = new MenuItem("Export Infrastructure", menu.exportItemImageResource);
```

Comment:
This method gets really long. Maybe you can put this initializing in a private method

Reply:
Yes, applied.